src/main/java/org/wildfly/security/auth/server/ServerAuthenticationContext.java

```diff
@@ -982,14 +980,7 @@ private void handleOne(final Callback[] callbacks, final int idx) throws IOExcep
                     handleOne(callbacks, idx + 1);
                 } else if (callback instanceof SSLCallback) {
                     SSLCallback sslCallback = (SSLCallback) callback;
-
-                    try {
-                        X509Certificate[] x509Certificates = X500.asX509CertificateArray(sslCallback.getSslSession().getPeerCertificates());
-                        verifyEvidence(new X509PeerCertificateChainEvidence(x509Certificates));
```

Comment:
This call to verifyEvidence is required for the EXTERNAL mechanism, i.e. that mechanism currently operates to use whatever identity has been previously authenticated.  

Remoting would only trigger this callback if EXTERNAL is selected, the verifyEvidence call would happen and then the EXTERNAL mech can go straight to AuthorizeCallback and use the previously authenticated identity.